io/ntriples/common/src/main/scala/org/w3/banana/io/NTriplesWriter.scala

```diff
@@ -24,26 +27,35 @@ class NTriplesWriter[Rdf <: RDF](implicit val ops:RDFOps[Rdf]) extends RDFWriter
    * @param node Rdf node
    * @return
    */
-  def node2Str(node:Rdf#Node): String = ops.foldNode(node)(
-    { case ops.URI(url) => "<" + url + ">"},
-    { case ops.BNode(id) => "_:" + id},
-    {
-      case ops.Literal((string, ops.URI(datatype), Some(lang))) => "\"" + string + "\"" + "@" + lang
-      case ops.Literal((string, ops.URI(datatype), None)) => "\"" + string + "\"" + "^^<" + datatype + ">"
+  def node2Str(node: Rdf#Node, base: Rdf#URI): String = ops.foldNode(node)(
+    uri => "<" + asString(uri, base) + ">",
```

Comment:
yes, the [asString function was redefined further up in the same file](https://github.com/read-write-web/banana-rdf/blob/ntriplesFixes/io/ntriples/common/src/main/scala/org/w3/banana/io/NTriplesWriter.scala#L40) which fixes https://github.com/w3c/banana-rdf/issues/239